### PR TITLE
CK Editor: Small fixes, Rake Tasks

### PR DESCRIPTION
**Pull request by nemaniarjun**
This PR has the final changes required for merging CK editor for production:

1. Changes to TOC generation
2. Changes to CSS for md-content h1 -> h2, h2 -> h3 ... 
3. Rake task to update all the existing challenge description headings, h1 -> h2, h2 -> h3 ... 
4. Update to the min-width of the editor to match that of the overview page.

### CODE REVIEW

**Comment by nemaniarjun** [comment]
@skbly7 @piotrekpasciak @alkeshrails Please review this PR from the review app as well.
I have ran the rake task and it seems to have worked as expected.

File: lib/tasks/challenges.rake [hunk 1]
```diff
@@ -8,13 +8,33 @@ namespace :challenges do
...
+          score_secondary_title:   challenge.score_secondary_title,
+          primary_sort_order_cd:   challenge.primary_sort_order_cd,
+          secondary_sort_order_cd: challenge.secondary_sort_order_cd
       )
 
       puts "##{challenge.id} Challenge - finished migration of score titles."
     end
   end
+
+  desc "Migrate description headings for all challenges (h1 -> h2, h2 -> h3, h3 -> h4)"
```
Comment (skbly7) [inline lib/tasks/challenges.rake]:
```ruby
  desc "Migrate description headings for all challenges (h1 -> h2, h2 -> h3, h3 -> h4)"
  task migrate_all_challenge_descriptions: :environment do
    puts "Migration for all challenge descriptions started"
    Challenge.all.find_each do |challenge|
      next if challenge.description.blank?

      puts "Migrating description for challenge #{challenge.challenge}"
      doc = Nokogiri::HTML.fragment(challenge.description)
      doc.css("h3").each{|h3| h3.name = "h4" }      
      doc.css("h2").each{|h2| h2.name = "h3" }
      doc.css("h1").each{|h1| h1.name = "h2" }
      challenge.description = doc.to_html
      challenge.save!
  end
end
```

Comment (skbly7) [reply lib/tasks/challenges.rake]: @piotrekpasciak can you check the above comment? I will run it on review apps for testing purposes

-- @nemaniarjun

Comment (piotrekpasciak) [reply lib/tasks/challenges.rake]: Code looks good, but I'm still recommending  to export this code to MigrateDescriptionService and writing extensive code coverage.

**Pull request merged**